sklearn/tree/_classes.py

```diff
@@ -1100,6 +1100,7 @@ def __sklearn_tags__(self):
         }
         tags.classifier_tags.multi_label = True
         tags.input_tags.allow_nan = allow_nan
+        tags.input_tags.sparse = True
```

Comment:
This could be set in the in the base class as well and for the decision tree regressor.

Reply:
done in 705c414